test/mock-settlement.js

```diff
@@ -0,0 +1,12 @@
+const { SFI_FIRST_PAYMENT: SFI_FIRST_PAYMENT_INVOICE_NUMBER } = require('./mock-components/mock-invoice-number')
+
+module.exports = {
```

Comment:
I think you can mock these individual parts into `mock-components/` folder

For e.g. I make us of `frn` so will only define it once :)